languages/javascript/exercises/concept/basics/basics.spec.js

```diff
@@ -0,0 +1,53 @@
+import {
+  EXPECTED_MINUTES_IN_OVEN,
+  remainingMinutesInOven,
+  preparationTimeInMinutes,
+  totalTimeInMinutes,
+} from './example'
+
+describe('basics', () => {
+  it('EXPECTED_MINUTES_IN_OVEN', () => {
+    expect(EXPECTED_MINUTES_IN_OVEN).toBe(40)
+  })
+
+  describe('remainingMinutesInOven', () => {
+    ;[
```

Comment:
Do we need a semicolon here?

Reply:
Autoformatting from prettier. It's probably to ensure that you can add anything on line 13/14, without it breaking.

Reply:
I see.